tests/compile/excvalcheck.h

```diff
@@ -1,12 +1,6 @@
-#ifdef __cplusplus
-extern "C" {
-#endif
```

Comment:
This is in test code. Why did you remove these?

Reply:
I am just adjusting existing tests to the new behavior.
With the old behavior, functions were declared as `extern` in C mode and `extern "C"` in C++ mode by Cython, and all declarations within a compilation unit must agree.
With the new behavior, functions are declared as `extern` in C mode and `extern "C++"` in C++ mode by Cython, so these `extern "C"` blocks make the compilation fail.